library/cwm/src/modules/CWM.rb

```diff
@@ -469,13 +469,6 @@ def ValidateMaps(widgets)
         else
           ["label", "widget"]
         end
-        if !Builtins.haskey(v, "no_help")
```

Comment:
OK, we have been quietly adding no_help to AbstractWidgets anyway, so dropping that for WidgetHash is not a big problem